ui/app/AppLayouts/Profile/LeftTab/Menu.qml

```diff
@@ -31,8 +30,8 @@ ScrollView {
                 active: profileMenu.profileCurrentIndex === modelData.id
                 Layout.fillWidth: true
                 width: profileMenu.width
-                onClicked: function () {
-                    profileMenu.profileCurrentIndex = modelData.id
+                onClicked: {
+                    Constants.currentMenuTab = modelData.id;
```

Comment:
Constants shouldn't be modified IMO.

Either you can put `currentMenuTab` in the AppMain.qml file or maybe we can create another Singleton called CurrentState or something if we want properties that can change.

Reply:
@jrainville putting currentMenuTab in the AppMain.qml file is not a good idea, how should the PrivateChatPopup access it? As I already said above, I could either use Utils or create another one for this property. In general terms, I would restructure the whole thing so no singleton would be required in that case, but this is a bigger task and not relevant to this one.

Reply:
> how should the PrivateChatPopup access it?

It can access it as a "global" property kinda. But if you feel like it's bad, we can put it somewhere else. I just feel like `Constants` should stay constant